WordPress/Classes/ViewRelated/Reader/ReaderMenuViewController.swift

```diff
@@ -84,9 +84,10 @@ import WordPressShared
             restorableSelectedIndexPath = defaultIndexPath
         }
 
-        cleanupStaleContent(removeAllTopics: false)
+        unflagInUseContent()
```

Comment:
I'm not too worried about it, but if you have a strong concern we could move this call to a `UIApplicationWillTerminate` notification observer.  I think it might makes things a bit less obvious but some extra documentation would help.   Thoughts?